scripts/lis_hyperv_platform/lisa_run.ps1

```diff
@@ -42,12 +42,34 @@ function Edit-XmlTest {
     popd
 }
 
+function Check-LisaExit {
+    param(
+        [int] $exitCode
+    )
+
+    $icaPath=".\TestResults\**\ica.log"
+    if ($exitCode) {
```

Comment:
to not indent too much code, as you do not have an else statement, you can just:
if (!ExitCode) { return }

continue with the unindented code
it makes the code more readable